Config.properties

```diff
@@ -0,0 +1,3 @@
+baseURL = https://www.olympus-ims.com/en/login/
+emailID = admin@abc.com
+password = Admin@123
```

Comment:
Password should not be added in .properties file

Reply:
By mistakenly I have pushed wrong file as I have already added all variables in .properties file
So I will delete this file from here

Reply:
remove password from config file